acceptance/Gemfile

```diff
@@ -1,6 +1,6 @@
 source ENV['GEM_SOURCE'] || "https://rubygems.org"
 
-gem "beaker", "~> 1.17"
+gem "beaker", "~> 2.23"
```

Comment:
Basically I'm trying to determine if you meant to pin the dependency to an explicit version or if you were okay with pessimistic versioning bumps.